### DISCUSSION TITLE
✅ 🔗 🚀 Feature Request: Allow to use custom icons in the Star Rating widget (Duplicate of #10529) - [ED-11941]

### DISCUSSION THREAD

**Discussion by knireis** [elementor/elementor#22045] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I try to use alternative icons in the rating widget. The widget does not have the option to choose an icon. It is only stars....

### Describe the solution you'd like

Be able to choose other icons than stars

### Describe alternatives you've considered

maybe with css but this seems quite complicated

### Additional context

This is very usefull when you need to specify for instance some characteristics of a product. As example the degree of softness, or the degree of durability and so on.

**Comment by nicholaszein** [comment]
Hello @knireis!

Thank you so much for your patience! 🙏 

We have great news! 🙌 

### 📢 We're happy to announce that the request you raised was resolved with the release of the new Rating widget in Elementor v3.17! 🥳 

With this new widget, you have more options to choose the number of rating points, custom icons, and styling options, and it is fully compatible with Google Rich snippets (updated) and fully accessible.

Your current Start Rating widgets will remain as they were before this update. But moving forward, only the new Rating widget will be available to add to your layouts.

If you are using the old Star Rating widget, we would recommend that you replace it with the new Rating widget.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂

**Reply by nicholaszein** [reply]
This request is a duplicate of #10529
